.github/workflows/contracts.yaml

```diff
@@ -99,7 +99,7 @@ jobs:
 
   contracts-deployment-testnet:
     needs: [contracts-build-and-test]
-    if: github.event_name == 'workflow_dispatch'
+    if: github.event_name == 'workflow_dispatch' && github.event.inputs.environment == 'goerli'
```

Comment:
Do we need to add `&& github.event.inputs.environment == 'goerli'`?
It's a manual workflow dispatch step, so maybe we could skip it?

Reply:
[8c0c77d](https://github.com/threshold-network/solidity-contracts/pull/114/commits/8c0c77df74013532e4f049ea994806ae22abad18)